src/engine/enginebuffer.cpp

```diff
@@ -41,8 +41,18 @@
 #include "engine/vinylcontrolcontrol.h"
 #endif
 
+namespace {
+
 const double kLinearScalerElipsis = 1.00058; // 2^(0.01/12): changes < 1 cent allows a linear scaler
-const int kSamplesPerFrame = 2; // Engine buffer uses Stereo frames only
+
+const SINT kSamplesPerFrame = 2; // Engine buffer uses Stereo frames only
+
+inline void assertValidPlayPosition(double playPosition) {
```

Comment:
assertPlayPositionAtFrameStart

We may also consider to make kSamplesPerFrame a parameter and move it to util/sample.h